public/javascripts/scripts.js

```diff
@@ -81,7 +82,10 @@ socket.on('car update', (car) => {
 });
 
 let addCarBtn = document.querySelector("#addCarBtn");
+let amountOfCars = document.querySelector("#amountOfCars");
 
 addCarBtn.addEventListener("click", () => {
-  fetch("/cars", { method: "POST" }).then(() => {});
+  for (let index = 0; index < amountOfCars.value; index++) {
```

Comment:
Dit verklaart volgens mij de memory overflow... Je kunt beter een request maken:

```javascript
fetch(`/cars?amountOfCars=${amountOfCars.value}`, { method: "POST" }).then(() => {});
```

En dan halen we deze parameter op op de server en starten `simulateCar()` meerdere malen.